cli/server.go

```diff
@@ -277,8 +279,8 @@ func server() *cobra.Command {
 				}
 			}
 
-			_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "access-url: %s\n", accessURL)
-			_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "provisioner-daemons: %d\n", provisionerDaemonCount)
+			_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "Create the following number of provisioner daemons on start: %d\n", provisionerDaemonCount)
```

Comment:
is this accurate?

Reply:
It is, but the action is done in the context of the `coder server` while the next `Access your...` is in the context of the user.

Reply:
@ammario wdyt: `Create the following number of provisioner daemons when running coder server` or similar?